easybuild/framework/easyconfig/default.py

```diff
@@ -137,7 +137,7 @@
     'dependencies': [[], "List of dependencies", DEPENDENCIES],
     'hiddendependencies': [[], "List of dependencies available as hidden modules", DEPENDENCIES],
     'osdependencies': [[], "OS dependencies that should be present on the system", DEPENDENCIES],
-
+    'system_modules': [[], "System module dependencies that should be present on the system", DEPENDENCIES],
```

Comment:
ok, I see...

well, I don't feel too strongly about it, so `external_dependencies` is fine by me too; up until now we haven't been using any `_` for `*dependencies` though, so that would make it `externaldependencies`...

or maybe `external_dependency_modules`?